### PR TITLE
feat(nuttx_image_cache): add tid for cache name

### PR DESCRIPTION
**Pull request by W-Mai**
### Description of the feature or fix

add tid for cache name

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: src/drivers/nuttx/lv_nuttx_image_cache.c [hunk 1]
```diff
@@ -30,6 +30,8 @@ typedef struct {
     uint8_t * mem;
     uint32_t mem_size;
 
+    char name[sizeof(HEAP_NAME) + 10];
```
Comment (XuNeo) [inline src/drivers/nuttx/lv_nuttx_image_cache.c]: Need to comment why `+ 10`.

Comment (W-Mai) [reply src/drivers/nuttx/lv_nuttx_image_cache.c]: experience value

**Review by XuNeo — DISMISSED** [review]

File: src/drivers/nuttx/lv_nuttx_image_cache.c [hunk 2]
```diff
@@ -30,6 +30,9 @@ typedef struct {
     uint8_t * mem;
     uint32_t mem_size;
 
+    char name[sizeof(HEAP_NAME) + 10]; /**< UINT8_MAX is a maximum of 3 bits long in decimal notation,
+                                            and with the [] on both sides, 10 bytes must be enough */
```
Comment (kisvegabor) [inline src/drivers/nuttx/lv_nuttx_image_cache.c]: I don't understand this comment :sweat_smile: 

Note that `sizeof("GImageCache")` return 4 on 32 bit systems. If this the intention `sizeof(const char *)` would be cleaner.

Comment (W-Mai) [reply src/drivers/nuttx/lv_nuttx_image_cache.c]: in fact, sizeof("GImageCache") is 12, a constant value of the string length with `\0` 🥹

Comment (XuNeo) [reply src/drivers/nuttx/lv_nuttx_image_cache.c]: The comment would be easier to understand say: 
"+10 characters to store task pid."

Comment (W-Mai) [reply src/drivers/nuttx/lv_nuttx_image_cache.c]: Done

**Review by XuNeo — APPROVED** [review]

File: src/drivers/nuttx/lv_nuttx_image_cache.c (hunk 2 above)
Comment (kisvegabor) [reply src/drivers/nuttx/lv_nuttx_image_cache.c]: > in fact, sizeof("GImageCache") is 12, a constant value of the string length with \0 🥹

Wow, black magic :smile: But you are right!

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**